src/main.c

```diff
@@ -359,6 +360,7 @@ printUsage(ENGINE* engine) {
   ENGINE_printLog(engine, "  -h --help           Show this screen.\n");
   ENGINE_printLog(engine, "  -r --record=<gif>   Record video to <gif>.\n");
   ENGINE_printLog(engine, "  -v --version        Show version.\n");
+  ENGINE_printLog(engine, "  -w --wren2cstring   Converts a Wren source to a C String (helps plugin dev).\n");
```

Comment:
An extra line should be added to show how this is used, under line 351
```
 352 |  ENGINE_printLog(engine, "  dome -w | --wren2cstring [--] sourceFile [moduleName] [destinationFile]\n");
```

I would change the description of the flag to "Converts a Wren source file to a C include file."